.evergreen.yml

```diff
@@ -241,32 +198,34 @@ functions:
       params:
         working_dir: src
         shell: bash
+        env:
+          <<: *compass-env
+          # TODO: npm_loglevel, debug, compass_distribution
         script: |
           set -e
-          source ~/compass_env.sh
+          eval $(node scripts/print-compass-env.js);
 
+          # TODO: move
           cat <<EOF_BUILD_SH > ~/compass_package.sh
           #! /usr/bin/env bash
+          # TODO: why?
           cd $(pwd)
```

Comment:
🤣 .. yeah as weird as this looks it makes sense, essentially is creating a script that switch immediately to the root of the build no matter where is called from afterwards.. i'm assuming it was necessary cause the directory at that point would have been different 🤷‍♂️

Reply:
surely pwd is always the current directory?

Reply:
@lerouxb Keep in mind that this is something that is being expanded when the script is being written to disk – i.e. `compass_package.sh` itself will contain something like `cd /data/mci/.../src` afterwards instead of a literal `$(pwd)` string

Reply:
I'm going to have to deal with that because I'm moving this to an external script.